default-value-interface/README.md

```diff
@@ -0,0 +1,24 @@
+# SuperTRUMP for Salesforce Default Value Interface Add-On
+
+This project is an extension of the SuperTRUMP for Salesforce managed package.
+https://appexchange.salesforce.com/appxListingDetail?listingId=a0N3000000B5JerEAF
+
+SuperTRUMP for Salesforce has a Default Value Mapping object that allows package subscribers to set default values in the SuperTRUMP interface. However, it's only possible to set static values, or refer to related fields to set the desired values. It's not possible to conditionally set default values or use other custom logic or dependencies to determine what default values should be used.
+
+SuperTRUMP for Salesforce exposes an interface in apex that package subscribers can use to create default values for the SuperTRUMP interface. Creating default values in apex allows a developer to use SOQL, custom logic, and other features available in apex to customize the default values to the needs of the organization. For example, creating an apex class that implements this interface could set default values based on the template of the quote being created.
+
```

Comment:
Can you add some additional comments about the customer needing to review and update the SuperTRUMPDefaultValues.cls that is provided. We want to make it explicit that this class is an example implementation and their developers should review it and make appropriate changes to satisfy the needs of the business.